jobApplicationTracker/Models/EmploymentType.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace jobApplicationTrackerApi.Models
+{
+    public class EmploymentType
+    {
+        [Key]
```

Comment:
I'd prefer floating API instead of attributes, since it's more flexible. But it's up to you

Reply:
Thanks for the suggestion, Larry! A floating API is new to me. Could you elaborate more on how to effectively implement it and integrate it into the project? I’d love to understand how it works in practice and how to best use it in this context. Your guidance would be greatly appreciated!